new-dti-website-redesign/src/components/TeamCard.tsx

```diff
@@ -0,0 +1,136 @@
+import { getColorClass, productLinks } from '@/utils/memberUtils';
+import Image from 'next/image';
+import Link from 'next/link';
+
+const MemberSummary = ({
+  user,
+  image,
+  enlarged = false
+}: {
+  user: IdolMember;
+  image: string;
+  enlarged?: boolean;
+}) => (
+  <>
+    <Image
+      src={image}
+      alt={`${user.firstName} ${user.lastName}`}
+      className="rounded-lg w-auto h-auto"
+      width={232}
+      height={232}
+    />
+    <div className="text-left">
+      {enlarged ? (
+        <h3>{`${user.firstName} ${user.lastName}`}</h3>
+      ) : (
+        <h6>{`${user.firstName} ${user.lastName}`}</h6>
+      )}
+      <p className={`text-${getColorClass(user.role)}`}>{user.roleDescription}</p>
+    </div>
+  </>
+);
+
+type MemberCardProps = {
+  user: IdolMember;
+  image: string;
+  selected: boolean;
+  onClick?: () => void;
+};
+
+export const MemberCard = ({ user, image, selected, onClick }: MemberCardProps) => {
+  const baseStyles =
+    'p-8 border border-border-1 flex flex-col gap-4 focusState hover:shadow-white hover:shadow-[0_-4px_8px_0]';
+
+  return (
+    <button
+      className={`${baseStyles} ${selected ? 'border-b-foreground-1' : ''}`}
+      onClick={onClick}
+    >
+      <MemberSummary user={user} image={image} />
+    </button>
+  );
+};
+
+type MemberDetailsProps = {
+  user: IdolMember;
+  image: string;
+};
+export const MemberDetailsCard = ({ user, image }: MemberDetailsProps) => {
+  const baseStyles = 'w-1/2 p-8 border border-border-1 flex flex-col';
+  return (
+    <div className="flex w-full">
+      <div className={`${baseStyles} gap-4`}>
+        <MemberSummary user={user} image={image} enlarged />
+      </div>
+      <div className={`${baseStyles} gap-8`}>
+        <div className="flex gap-8">
+          <div className="w-1/2">
+            <p className="text-foreground-3">Graduating</p>
+            <p>{user.graduation}</p>
+          </div>
+          <div className="w-1/2">
+            <p className="text-foreground-3">Major</p>
+            <p>{`${user.major}${user.doubleMajor ? ` & ${user.doubleMajor}` : ''}`}</p>
+          </div>
+        </div>
+        <div className="flex gap-8">
+          <div className="w-1/2">
+            <p className="text-foreground-3">Hometown</p>
+            <p>{user.hometown}</p>
+          </div>
+          <div className="w-1/2">
+            <p className="text-foreground-3">Subteam</p>
+            {user.subteams[0] in productLinks ? (
+              <Link
+                className="flex gap-2 items-center"
```

Comment:
```suggestion
                className="flex gap-2 items-center focusState w-fit"
```

for custom focus state and also to make the link not span the entire width of the col